lib/screens/my_profile.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+
+class my_profile extends StatelessWidget{
+  const my_profile({
+  required this.text,
+  required this.icon,
+  required this.iconColor,
+  super.key});
+  final String text;
+  final Icon icon;
+    final Color iconColor; 
+  
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.center,
+      children: [
+        Padding(
+          padding: const EdgeInsets.only(left: 10.0),
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Container(
+                width: MediaQuery.of(context).size.width / 1.1,
```

Comment:
The width of the container is set to 1.1, which might not be responsive on all devices. Consider using MediaQuery.of(context).size.width * 0.9 or a similar fraction to ensure responsiveness across various screen sizes.